api/api.go

```diff
@@ -69,61 +62,49 @@ func (s *Settings) Validate() error {
 	return nil
 }
 
-// New returns the minimum set of REST APIs needed for the Service Manager
-func New(ctx context.Context, repository storage.Repository, settings *Settings, encrypter security.Encrypter) (*web.API, error) {
+// NewInterceptableRepository returns the minimum set of REST APIs needed for the Service Manager
```

Comment:
Fix the comment - should not be **NewIntereceptableRepository**, should be just **New**